lib/project_types/script/forms/create.rb

```diff
@@ -31,13 +31,7 @@ def valid_name
       end
 
       def ask_language
-        if language
-          if Layers::Application::ExtensionPoints.supported_language?(type: extension_point, language: language)
-            return language.downcase
-          else
-            raise Errors::InvalidLanguageError.new(language, extension_point)
-          end
-        end
```

Comment:
I am not seeing where you moved it to.  Alternatively, you can also just pass in the exact list of supported languages in this function, and let it prompt the user to choose one.  Similarly for the EP.

Reply:
It will still prompt the user for the language like EP, as long as the user didn't specify a `language` flag. The section I moved is just validating that the selected language is supported. We should validate the language during each command to make sure our project is setup properly so the ScriptProjectRepository validates the language here:
https://github.com/Shopify/shopify-app-cli/blob/533e5f39d39ae571f6dc4d530dc514463a8f6a2f/lib/project_types/script/layers/infrastructure/script_project_repository.rb#L100-L101

From the end-user's perspective, nothing will change. Heres a screenshot showing selecting a valid language through a flag, selecting an invalid language through a flag, and being prompted to select a language instead of using a flag.

<details>
<img src=https://user-images.githubusercontent.com/28009669/115921890-2201a380-a44a-11eb-9980-b3aa1d5a6e79.png />

</details>